theano/sparse/basic.py

```diff
@@ -233,9 +233,7 @@ def conv_op(*inputs):
         out = op(*ipt)
         return oconv(out)
 
-    seed_rng()
-    rng = numpy.random
-    T.verify_grad(conv_op, dpt, 2, rng, *args, **kwargs)
```

Comment:
Is the seed_rng() actually doing anything though? It seems that it's fetching a seed from config.unittests.rseed, but is that actually non-None when someone calls verify_grad_sparse? I'm not clear who the expected callers for this function are.